### ISSUE TITLE
expected Tensor as element 2 in argument 0, but got int

### ISSUE THREAD

**Issue by zhoupeng0418** [sunsmarterjie/GOLD_NAS#1]
When i using 'python ./cifar_search/train_search.py \\',after a few hours,an error  shows :"expected Tensor as element 2 in argument 0, but got int" ,it comes from 'model_search.py'-----'class Cell(nn.Module)'----'return torch.cat(states[2:], dim=1)'. 

Is this result normal? If not, can you give me some advice?

**Comment by sunsmarterjie** [comment]
This is a bug, I will update model_search.py.

**Comment by zhoupeng0418** [comment]
thanks for you reply.the bug may be cause by "weights" in mode_search.py. it return "nal " when search process
